### PR TITLE
Create blog post for release v1.39

### PR DESCRIPTION
**Pull request by DNin01**
This is the blog post that will go along with [Scratch Addons v1.39.0](https://github.com/ScratchAddons/ScratchAddons/releases/tag/v1.39.0).

### CODE REVIEW

File: content/blog/v1-39-released.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: What's new in Scratch Addons v1.39
+description: Handle asset import conflicts, delete all but one asset, and more.
+date: null # TBD
```
Comment (DNin01) [inline content/blog/v1-39-released.md:4]: Specify this before merging.

File: content/blog/v1-39-released.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+author: DNin01
+---
+
+# What's new in Scratch Addons v1.39.0
```
Comment (Hans5958) [inline content/blog/v1-39-released.md:8]: Remove this. Title goes to the front matter.

```suggestion
```

File: content/blog/v1-39-released.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+Plenty of new features have been added in this update to improve your productivity on Scratch.
+
+Let's start with the new addons.
+
+### 📄 Choose what to do with conflicting asset names
```
Comment (Hans5958) [inline content/blog/v1-39-released.md:14]: Use the second-level heading as it is for semantics, not for heading font size reasons (also true to later headings).

```suggestion
## 📄 Choose what to do with conflicting asset names
```

**Review by Hans5958 — CHANGES_REQUESTED** [review]

File: content/blog/v1-39-released.md (hunk 3 above)
Comment (Hans5958) [reply content/blog/v1-39-released.md:14]: I have to fix this with the previous posts later. For this post, this can be done now.

**Comment by DNin01** [comment]
All good now, @Hans5958?

**Comment by WorldLanguages** [comment]
@Hans5958 Blocking merge due to those things is a bit harsh 😅 You can also review without going red

File: content/blog/v1-39-released.md (hunk 1 above)
Comment (WorldLanguages) [reply content/blog/v1-39-released.md:4]:
```suggestion
date: 2024-08-31
```

**Review by Hans5958 — APPROVED** [review]

**Pull request merged**